includes/Traits/Login_Registration.php

```diff
@@ -1575,23 +1575,23 @@ public function eael_extra_user_profile_fields( $user ){ ?>
 		<?php $eael_custom_profile_fields_image = $this->get_eael_custom_profile_fields('image'); ?>
 
 		<?php //if ( count( $eael_custom_profile_fields_text ) || count( $eael_custom_profile_fields_image ) ): ?>
-		<h3><?php _e("EA Login | Register Form", "blank"); ?></h3>
+		<h3><?php esc_html_e('EA Login | Register Form', 'essential-addons-for-elementor-lite'); ?></h3>
 		<?php // endif; ?>
 
 		<table class="form-table">
 		<tr>
-				<th><label for="eael_phone_number"><?php _e("Phone"); ?></label></th>
+				<th><label for="eael_phone_number"><?php esc_html_e('Phone', 'essential-addons-for-elementor-lite'); ?></label></th>
 				<td>
 					<input type="text" name="eael_phone_number" id="eael_phone_number" value="<?php echo esc_attr( get_the_author_meta( 'eael_phone_number', $user->ID ) ); ?>" class="regular-text" /><br />
-					<p class="description"><?php esc_html_e("Please enter your phone number."); ?></p>
+					<p class="description"><?php esc_html_e('Please enter your phone number.', 'essential-addons-for-elementor-lite'); ?></p>
 				</td>
 			</tr>
 		<?php
 		if( count( $eael_custom_profile_fields_text ) ) :
 			foreach( $eael_custom_profile_fields_text as $eael_custom_profile_field_text_key => $eael_custom_profile_field_value ) :
 		?>
 			<tr>
-				<th><label for="<?php echo esc_attr( $eael_custom_profile_field_text_key ); ?>"><?php _e( esc_html( $eael_custom_profile_field_value ) ); ?></label></th>
+				<th><label for="<?php echo esc_attr( $eael_custom_profile_field_text_key ); ?>"><?php esc_html( $eael_custom_profile_field_value ); ?></label></th>
```

Comment:
Need to echo.

`echo esc_html` or `esc_html_e` 

Also add domain if translation is used.